content/boards/unsupported/arduino-uno-r4/index.md

```
@@ -0,0 +1,23 @@
+latter makes it incompatible with MobiFlight firmware. Arduino's advertised sketch compatibility
+applies only to Arduino sketches the user builds in the Arduino IDE, not pre-made firmware images
+like MobiFlight.
+
+{{< cards >}}
+
+{{< card title="Arduino Uno R4 Minima" subtitle="Renesas RA4M1 microcontroller" tag="Unsupported" tagType="error" image="card-images/boards/arduino-uno-r4-minima.png" method="Resize" options="600x q80 webp" >}}
+{{< card title="Arduino Uno R4 Wi-Fi" subtitle="Renesas RA4M1 microcontroller" tag="Unsupported" tagType="error" image="card-images/boards/arduino-uno-r4-wifi.png" method="Resize" options="600x q80 webp" >}}
+
+{{</ cards >}}
+
+The Arduino Uno R4 comes in two variants, Minima and WiFi. Both are incompatible with MobiFlight firmware.
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)